.config/dotnet-tools.json

```diff
@@ -3,10 +3,11 @@
   "isRoot": true,
   "tools": {
     "docfx": {
-      "version": "2.75.3",
+      "version": "2.77.0",
       "commands": [
         "docfx"
-      ]
+      ],
+      "rollForward": false
```

Comment:
I'm wondering, what is "rollForward" doing and why set it to false? Just wondering why we need it in here, since docfx was working okay before this.

Reply:
@ncguilbeault this was added automatically when I updated `docfx`. I have tried with a clean manifest and install following the [NuGet instructions](https://www.nuget.org/packages/docfx) and it still gets added.

Understanding the default behavior is surprisingly complicated and I wasn't able to find good references on this. There were also a number of changes merged into `dotnet` to make this an overridable setting (e.g. see https://github.com/dotnet/sdk/pull/37231). I don't have time to look more into this right now, but if you want we can remove it.

Reply:
Strange but yeah, I also found that the documentation for this is lacking. My takeaway from [this article](https://learn.microsoft.com/en-us/dotnet/core/versions/selection) is that its related to how dotnet tools select which .NET version when the tool targets an earlier version than one that is installed. I'm still not sure why this flag wasn't there before with the previous version of docfx. Lets just leave it in there going forward since it shouldn't have any impact on how we build the docs.